torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -314,3 +355,46 @@ def resized_crop_image_pil(
 ) -> PIL.Image.Image:
     img = crop_image_pil(img, top, left, height, width)
     return resize_image_pil(img, size, interpolation=interpolation)
+
+
+def zoom_out_image_tensor(
```

Comment:
Hmmm.. Does it make sense to create a function equivalent for every possible transform? One can argue that the functional should be kept simple. Perhaps that's something we can leave in the Transform. Thoughts?

Reply:
You are probably right. These three kernels are basically thin wrappers around the padding kernels. In fact, everything here should probably be supported by the padding kernels directly. If we don't want to go there right now, I can also move this back onto the transform. This will make the implementation there more elaborate than just a plain if / else, but I would be ok with that for now.

Reply:
Yeah I agree with that approach. We probably need a definition of what qualifies to be added in the transforms and what should be its own kernel. I suspect that there are other kernels that shouldn't be in functional and instead should move in the Transforms (perhaps Mixup/cutmix). We should review later and somehow try to formalize in the design doc.